public/src/pages/StudentPage.tsx

```diff
@@ -302,7 +314,7 @@ export class StudentPage extends ViewPage {
     private selectGroupAssignment(labId: number) {
         if (this.selectedUserGroupCourse) {
             // TODO: Be carefull not to return anything that sould not be able to be returned
```

Comment:
Is this TODO useful? (I know it is probably old). Maybe it should be fixed or written as comment.

Reply:
I have no idea what it even refers to.

Reply:
Seems reasonable to remove these TODOs since these functions don't return anything...

Reply:
Already done.